test/mobx.js

```diff
@@ -153,11 +162,20 @@ describe('usage with mobx should generally work', () => {
       },
     })
     expect(service).to.have.callCount(1)
-    expect(reactor).to.have.callCount(3)
+    expect(reactor).to.have.callCount(18)
```

Comment:
This number seems really high

Reply:
Since the properties are extended from the beginning, and the reaction is triggered with any property change (since we're doing toJS), this number is correct.